src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -578,56 +576,48 @@ static String normalizeServer(String server) { /*package scope for testing*/
 
     @Restricted(NoExternalUse.class)
     public ApplicationContext createApplicationContext(LDAPSecurityRealm realm) {
-        DefaultInitialDirContextFactory initialDirContextFactory = new DefaultInitialDirContextFactory(getLDAPURL());
+        DefaultSpringSecurityContextSource contextSource = new DefaultSpringSecurityContextSource(getLDAPURL());
         if (getManagerDN() != null) {
-            initialDirContextFactory.setManagerDn(getManagerDN());
-            initialDirContextFactory.setManagerPassword(getManagerPassword());
+            contextSource.setUserDn(getManagerDN());
+            contextSource.setPassword(getManagerPassword());
```

Comment:
Inferred from `LdapAuthenticationProviderConfigurer` that this is the equivalent.